uber/models.py

```diff
@@ -700,13 +700,15 @@ def delete_from_group(self, attendee, group):
             self.delete(attendee)
             group.attendees.remove(attendee)
 
-        def assign_badges(self, group, new_badge_count, new_badge_type=None, **extra_create_args):
-            new_badge_type = new_badge_type or c.ATTENDEE_BADGE
```

Comment:
I guess there's no way to know what kind of ribbon things should be in, say, prereg payment pages, and we don't want our code to just add ribbons when attendees are doing normal things like adding new badges.